contact.html

```diff
@@ -64,7 +64,7 @@
     <nav class="navbar navbar-expand-lg navbar-light bg-transparent style-5">
         <div class="container-fluid">
             <a class="navbar-brand" href="#">

```

Comment:
The logo should link back to the index page. Instead it will take you to the top of the contact page.